### PR TITLE
DR2-1592 Delete old lambdas and queues.

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

**Review by techncl — COMMENTED** [review]
Looks good! Just some minor comments

File: templates/dynamo/initial_last_updated_item.json.tpl [hunk 1]
```diff
@@ -3,6 +3,6 @@
     "S": "LastPolled"
   },
   "datetime": {
-    "S": "1970-01-01T00:00Z"
+    "S": "2024-04-24T10:14:57Z"
```
[the remark this answers is not in the document]
Comment (MancunianSam) [reply templates/dynamo/initial_last_updated_item.json.tpl:6]: Ah, I didn't mean to leave it like that. I changed it to stop terraform reverting the date when I recreated the table and forgot to change it back. I've reverted this now.

File: templates/logs/ingest_dashboard.json.tpl [hunk 2]
```diff
@@ -7,7 +7,7 @@
...
+        "query": "SOURCE '/aws/lambda/${environment}-dr2-ingest-parsed-court-document-event-handler' | SOURCE '/aws/lambda/${environment}-dr2-ingest-mapper' | SOURCE '/aws/lambda/${environment}-dr2-ingest-asset-opex-creator' | SOURCE '/aws/lambda/${environment}-dr2-ingest-folder-opex-creator' | SOURCE '/aws/lambda/${environment}-dr2-ingest-parent-folder-opex-creator' | SOURCE '/aws/lambda/${environment}-dr2-ingest-start-workflow' | SOURCE '/aws/lambda/${environment}-dr2-ingest-upsert-archive-folders' | fields @timestamp, message, error.message, log.level, @logStream\n| filter log.level == \"ERROR\"\n| sort @timestamp, batchRef desc\n| limit 20",
```
[the remark this answers is not in the document]
Comment (MancunianSam) [reply templates/logs/ingest_dashboard.json.tpl:10]: That's a good idea, thanks. I forgot about this. I only remembered when I was updating the confluence pages with screenshots and I got a load of errors :smile: 

I've gone with point 2 here. I've added an array of the lambdas we want to add to the dashboard and joined them together to make the source string.

I've added the two new ones to this array and added a line to the README about adding new lambdas to it.

**Review by techncl — APPROVED** [review]
👍

**Pull request merged**